### PR TITLE
Migrate video classification example to keras 3.0

### PR DESCRIPTION
**Pull request by GuichardVictor**
This PR migrates the video classification example to keras 3.0 as requested in [keras-team/keras-cv#2211](https://github.com/keras-team/keras-cv/issues/2211)

[Colab](https://colab.research.google.com/drive/1iW35I2w9GsdYQwzMVuwDN3pdXOEQ8L4B?usp=sharing)

### CODE REVIEW

File: examples/vision/video_classification.py [hunk 1]
```diff
@@ -47,18 +47,19 @@
 """
 ## Setup
 """
+import os
+
+os.environ["KERAS_BACKEND"] = "tensorflow"
```
Comment (fchollet) [inline examples/vision/video_classification.py:52]: What is preventing is from running with other backends?

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Review by fchollet — APPROVED** [review]
LGTM, thank you! Please generate the `md` and `ipynb` files.

**Comment by GuichardVictor** [comment]
Thanks you @divyashreepathihalli  for explaining the process to generate the .md and .ipynb files. Will do it asap.

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**